lib/controllers/share.js

```diff
@@ -137,15 +137,15 @@ function resolveS3(serviceOptions, id) {
 }
 
 
-module.exports = function(shareUrlPrefixes, newShareUrlPrefix, hostName, port) {
+module.exports = function(shareUrlPrefixes, newShareUrlPrefix, shareMaxRequestSize, hostName, port) {
```

Comment:
I think an options object would be better here, after the port. Lest we continue to increase the # of params here.
e.g. `shareOptions` with the `shareMaxRequestSize` as a key

Reply:
Yep I agree